.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Terminus Plugin Example
+on: push
+jobs:
+  checkout_lint:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
+    name: Checkout & lint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Save repo content as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: full-workspace
+          path: ${{ github.workspace }}
+      - name: Full Composer Install
+        run: composer install
+      - name: Install test tools
+        run: composer install-tools
+      - name: Validate Code
+        run: composer cs
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        php-versions: ['7.4', '8.0']
+    container:
+      image: pantheonpublic/terminus-plugin-test:4.x-php${{ matrix.php-versions }}
+    name: Test (PHP ${{ matrix.php-versions }})
+    needs: [ checkout_lint ]
+    defaults:
+      run:
+        shell: bash
+        working-directory: /terminus-plugin-test/plugin
+    env:
+      TERMINUS_TOKEN: ${{ secrets.TERMINUS_TOKEN }}
+      TERMINUS_SITE: ${{ secrets.TERMINUS_SITE }}
+      BASH_ENV: ~/.bashrc
+      HOME: /home/tester
+    steps:
+      - name: Download repo content from artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: full-workspace
+          path: /terminus-plugin-test/plugin
+      - name: Setup environment
+        run: chmod +x ./.ci/set-up-globals.sh && ./.ci/set-up-globals.sh
```

Comment:
Did you just copy this from Build Tools? I don't see the set-up-globals script.

Reply:
The directory was renamed from `.circleci` to `.ci`.  To answer your question, I copied from terminus-rsync-plugin.